### PR TITLE
Move Tags component to the slot belowContent

### PR DESCRIPTION
**Pull request by wesleybl**
Ref: https://github.com/plone/volto/pull/4381#issuecomment-2276168262

### CODE REVIEW

**Comment by wesleybl** [comment]
I removed showTags, but I don't know if it's the best option. It might make it easier to remove the tags.

CC @davisagli

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Add a little MyST markup to the news.

Otherwise this looks OK to me, but one more person should give a technical review.

**Review by davisagli — COMMENTED** [review]
I think this makes sense for Volto 18. Removing the showTags setting and changing the parameters expected by the Tags component are breaking changes that can't be backported.

**Review by erral — CHANGES_REQUESTED** [review]
Here you are changing the internal contract or the Tags component, which needs proper Upgrade Notes.

If it isn't completely necessary I wouldn't change the component itself, I would just do the slots thing.

**Comment by davisagli** [comment]
> If it isn't completely necessary I wouldn't change the component itself, I would just do the slots thing.

@erral The change is necessary in order to accept the parameters that are passed to slot components. (They get `content`, not `tags`.)

**Comment by wesleybl** [comment]
@erral @davisagli I changed the news to breaking and put a note in the update guide.

**Review by erral — APPROVED** [review]

**Comment by davisagli** [comment]
@wesleybl On second thought, I think we should keep the `showTags` setting. It's very easy to check it (inside the Tags component) and makes it easier for someone who is upgrading a site that disabled the tags (so they don't need to figure out how to override the slot component).

**Comment by wesleybl** [comment]
@davisagli I brought the `showTags` setting back.

File: packages/volto/news/6269.breaking [hunk 1]
```diff
@@ -0,0 +1 @@
+Move Tags component to the slot `belowContent`. @wesleybl
```
Comment (davisagli) [inline packages/volto/news/6269.breaking:1]:
```suggestion
Move `Tags` component to the slot `belowContent`. @wesleybl
```

**Review by davisagli — APPROVED** [review]

**Pull request merged**